susemanager-utils/susemanager-sls/susemanager-sls.changes.joyetamdk.bsc1232562

```diff
@@ -0,0 +1 @@
+- Enable venv-salt-minion disable salt-minion service based on os
```

Comment:
The changelog needs to mention the bug id and is target at the end-user. I think the current wording is not very clear for end-users. How about the following?
```suggestion
- Correctly switch to Salt Bundle on transactional systems
  (bsc#1232562)
```